base_agent/documents/logical_form_specification/human_give_command.md

```diff
@@ -287,9 +287,9 @@ where `has_x` can be : `has_block_type`, `has_name`, `has_size`, `has_orientatio
   "yaw_pitch": span,
   "yaw": span,
   "pitch": span,
-  "relative_yaw" = {"angle": -360, -180, -135, -90, -45, 45, 90, 135, 180, 360 
+  "relative_yaw" = {"fixed_value": -360 / -180 / -135 / -90 / -45 / 45 / 90 / 135 / 180 / 360, 
```

Comment:
same comment above- I assume this means xor, but comma is used differently than previous

Reply:
Same reply as above.